docs/UserGuide.md

```diff
@@ -40,95 +50,163 @@ Format: `help`
-> * Persons matching at least one keyword will be returned (i.e. `OR` search).
-    e.g. `Hans` will match `Hans Bo`
+For this kind of task, the parameter TIME is in the format START-END.
 
 Examples: 
-* `find John`<br>
-  Returns `John Doe` but not `john`
-* `find Betsy Tim John`<br>
-  Returns Any person having names `Betsy`, `Tim`, or `John`
+* `add dad's birthday celebration, jul 10, 1900-2100, high`
+* `add company's D&D, sun, 1900-2200`
+
+###### Recurring
+To make tasks repeating, simply  add the FREQUENCY parameter at the back.
 
+Format: `add TASK, [DATE], [TIME], [PRIORITY], [FREQUENCY]`
+
+Examples: 
+* `add water the plants, 0800, high, daily`
+* `add facial appointment, jul 10, med, monthly`
+
+#### Listing all tasks : `list`
+Shows a list of all tasks in a particular day.<br>
+Format: `list DATE`
+
+#### Searching tasks by keywords: `search`
+Searches and lists all tasks that contain the specified keyword.<br>
+Format: `search KEYWORD`<br>
+
+> * The search is not case sensitive.
+> * The order of the keywords does not matter. e.g. `buy milk oct 21` will match `oct 21 buy milk`
+> * Only full words will be matched e.g. `buy milk` will not match `buy cow`
+> * Tasks matching keywords will be returned
+
+Examples: 
+* `search oct 1`<br>
+  Lists all tasks that have the date `oct 1`.
+* `search milk`<br>
+  Lists the all the tasks with names containing the word `milk` if any can be found.
+  
 #### Deleting a person : `delete`
-Deletes the specified person from the address book. Irreversible.<br>
-Format: `delete INDEX`
+Deletes the specified task from the list. Irreversible.<br>
+Format: `delete INDEX...`
 
-> Deletes the person at the specified `INDEX`. 
+> Deletes the task at the specified `INDEX`. 
   The index refers to the index number shown in the most recent listing.<br>
   The index **must be a positive integer** 1, 2, 3, ...
 
 Examples: 
-* `list`<br>
-  `delete 2`<br>
-  Deletes the 2nd person in the address book.
-* `find Betsy`<br> 
+* `delete 2`<br>
+  Deletes the task indexed number second in the current seen list.
+* `search buy milk`<br> 
   `delete 1`<br>
-  Deletes the 1st person in the results of the `find` command.
+  Deletes the task indexed first in the results of the `search` command.
 
-#### Select a person : `select`
-Selects the person identified by the index number used in the last person listing.<br>
-Format: `select INDEX`
+To delete more than one task, simply add in the other indexes separated with a comma. <br>
 
-> Selects the person and loads the Google search page the person at the specified `INDEX`. 
-  The index refers to the index number shown in the most recent listing.<br>
-  The index **must be a positive integer** 1, 2, 3, ...
+Example:<br>
+* `delete 1, 2, 3`
+* `search oct 1`
+  `done 2, 3`
+  
+  
+#### Undo : `undo`
+Reverts the most recent action.<br>
+Format: `undo`
+
+#### Editing a task : `edit`
+Edits task on the displayed list.<br>
+Format: `edit INDEX PARAMETER NEW_INFORMATION`
```

Comment:
can we edit more than one parameter at once? if so, how? if not, pls mention that in this document.